doc-content/enterprise-only/installation/kie-server-thread-efficiency.adoc

```diff
@@ -0,0 +1,20 @@
+[id='kie-server-thread-efficiency.adoc_{context}']
+= Thread Efficiency
+
+To ensure the optimal number of threads used, set the threading system properties to the number of CPUs in the system + 1.
+
+. In your Red Hat Process Automation Manager installation directory, navigate to the standalone-full.xml file. For example, if you use a Red Hat JBoss EAP installation for Red Hat Process Automation Manager, go to $EAP_HOME/standalone/configuration/standalone-full.xml.
```

Comment:
Red Hat Process Automation Manager = {PRODUCT} If this is for DM as well, that variable updates for the DM book.
standalone-full.xml - code. Red Hat JBoss EAP = {EAP}.
go to = navigate to 
$EAP_HOME/standalone/configuration/standalone-full.xml should be code.

Reply:
Look through this whole section and make file names, paths, properties, etc code.